packages/curve-ui-kit/src/shared/ui/MarketDetails/index.tsx

```diff
@@ -71,6 +71,8 @@ export const MarketDetails = ({
       ? `${formatLiquidity(availableLiquidity.value)}/${formatLiquidity(availableLiquidity.max)}`
       : undefined
 
+  console.info('utilizationBreakdown', utilizationBreakdown)
```

Comment:
Please use console.log for debugging so we don't forget it
```suggestion
  console.log('utilizationBreakdown', utilizationBreakdown)
```